JSDemos/Demos/ColorBox/Overview/React/App.js

```diff
@@ -8,90 +8,84 @@ const readOnlyLabel = { 'aria-label': 'Read only' };
 const disabledLabel = { 'aria-label': 'Disabled' };
 const eventHandlingLabel = { 'aria-label': 'Event Handling' };
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
+function App() {
+  const [color, setColor] = useState('#f05b41');
 
-    this.state = { color: '#f05b41' };
-    this.handleColorChange = ({ value }) => this.setState({ color: value });
-  }
+  const handleColorChange = useCallback(({ value }) => {
+    setColor(value);
+  }, []);
 
-  render() {
-    return (
-      <React.Fragment>
-        <div className="form">
-          <div className="dx-fieldset">
-            <div className="dx-field">
-              <div className="dx-field-label">Default mode</div>
-              <div className="dx-field-value">
-                <ColorBox
-                  defaultValue="#f05b41"
-                  inputAttr={defaultModeLabel}
-                />
-              </div>
+  return (
+    <React.Fragment>
+      <div className="form">
+        <div className="dx-fieldset">
+          <div className="dx-field">
+            <div className="dx-field-label">Default mode</div>
+            <div className="dx-field-value">
+              <ColorBox defaultValue="#f05b41" inputAttr={defaultModeLabel} />
             </div>
-            <div className="dx-field">
-              <div className="dx-field-label">With alpha channel editing</div>
-              <div className="dx-field-value">
-                <ColorBox
-                  defaultValue="#f05b41"
-                  editAlphaChannel={true}
-                  inputAttr={alphaChannelLabel}
-                />
-              </div>
+          </div>
+          <div className="dx-field">
+            <div className="dx-field-label">With alpha channel editing</div>
+            <div className="dx-field-value">
+              <ColorBox
+                defaultValue="#f05b41"
+                editAlphaChannel={true}
```

Comment:
The `true` is not necessary to specify. Is this supports in demos?

Reply:
yes, it's handled by react